src/main/java/seedu/loyaltylift/storage/JsonAdaptedCustomer.java

```diff
@@ -119,7 +125,16 @@ public Customer toModelType() throws IllegalValueException {
         final Address modelAddress = new Address(address);
 
         final Set<Tag> modelTags = new HashSet<>(customerTags);
-        return new Customer(modelCustomerType, modelName, modelPhone, modelEmail, modelAddress, modelTags);
+
+        if (points == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Points.class.getSimpleName()));
+        }
+        if (!Address.isValidAddress(address)) {
+            throw new IllegalValueException(Address.MESSAGE_CONSTRAINTS);
+        }
```

Comment:
Should be checking for `Points` here right?

Reply:
yes, thanks for catching that bug, my bad